metaflow/plugins/list_cli.py

```diff
@@ -0,0 +1,69 @@
+from metaflow._vendor import click
+from metaflow import Flow
+from metaflow.exception import MetaflowNotFound
+import json
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group(help="List objects pertaining to your flow.")
+def list():
+    pass
+
+
+def _fetch_runs(flow_name, num_runs):
+    counter = 1
+    try:
+        flow = Flow(flow_name)
+    except MetaflowNotFound:
+        flow = None
+    run_list = []
+    if flow:
+        for run in Flow(flow_name).runs():
+            if counter > num_runs:
+                break
+            counter += 1
+            run_list.append(
+                dict(
+                    created=str(run.created_at),
+                    name=flow_name,
+                    id=run.id,
+                    status=run.successful,
+                    finished=run.finished,
+                )
+            )
+    return run_list
+
+
+@click.option(
+    "--num-runs",
+    default=10,
+    help="Number of runs to show.",
+)
+@click.option(
+    "--as-json",
+    default=False,
+    is_flag=True,
+    help="Print run list as a JSON object",
+)
+@click.option(
+    "--file",
+    default=None,
+    help="Save the run list to file as json.",
```

Comment:
The code makes it look like if you don't specify anything it doesn't print as json nor does it output to a file? There doesn't seem to also respect the quiet flag.

Reply:
Great catch, I accidentally forgot to add that back in while refactoring!  I will fix this!

Reply:
Ok I fixed this